Remora.Discord.Commands/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -112,7 +114,11 @@ public static IServiceCollection AddDiscordCommands
         );
 
         serviceCollection.AddCommands();
-        serviceCollection.AddCommandResponder();
+
```

Comment:
This should also conditionally exclude `CommandInteractionResponder`, since both handle incoming commands. Optionally, add a second parameter for that responder as well.

Reply:
Fair enough, thanks for the insight. ❤️

Reply:
I think you moved the wrong responder :P AutocompleteResponder should always be registered - it is `AddInteractionResponder` you want to conditionally add.